service/dap/server.go

```diff
@@ -1539,11 +1539,7 @@ func (s *Server) onStackTraceRequest(request *dap.StackTraceRequest) {
 	// Determine if the goroutine is a system goroutine.
 	// TODO(suzmue): Use the System() method defined in: https://github.com/go-delve/delve/pull/2504
```

Comment:
This TODO comment should now be taken out.

Reply:
Done.